source/common/protobuf/utility.h

```diff
@@ -310,6 +310,15 @@ class DurationUtil {
    * @throw OutOfRangeException when duration is out-of-range.
    */
   static uint64_t durationToSeconds(const ProtobufWkt::Duration& duration);
+
+  /**
+   * Writes a time_point<system_clock> (SystemTime) to a protobuf Timestamp, by way of time_t.
+   * @param system_clock_time the time to write
+   * @param timestamp a pointer to the mutable protobuf member to be written into.
+   */
+  static void
+  writeSystemClockTime(const std::chrono::time_point<std::chrono::system_clock> system_clock_time,
```

Comment:
nit: why not use the `SystemTime` typedef here?

Reply:
Oops, good catch.